modules/swagger-codegen/src/main/resources/ruby/gemspec.mustache

```diff
@@ -17,6 +17,7 @@ Gem::Specification.new do |s|
   s.summary     = "{{gemSummary}}{{^gemSummary}}{{{appName}}} Ruby Gem{{/gemSummary}}"
   s.description = "{{gemDescription}}{{^gemDescription}}{{{appDescription}}}{{^appDescription}}{{{appName}}} Ruby Gem{{/appDescription}}{{/gemDescription}}"
   s.license     = "{{gemLicense}}{{^gemLicense}}{{{licenseInfo}}}{{^licenseInfo}}Apache 2.0{{/licenseInfo}}{{/gemLicense}}"
+  s.required_ruby_version = "{{gemRequiredRubyVersion}}{{^gemRequiredRubyVersion}}>= 1.9{{/gemRequiredRubyVersion}}"
```

Comment:
@cliffano similar to `gemLicense`, I think you will also need to add `gemRequiredRubyVersion` to RubyClientCodegen.java:

https://github.com/swagger-api/swagger-codegen/blob/e044f9d9bf50efb40beb7f623a8e21e8a134bc84/modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/RubyClientCodegen.java#L32